src/components/SearchCart/SearchCart.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const SearchCart = ({item}) => {
```

Comment:
Use export instead of default one, as we are not using default export in components